doc/source/development/test_writing.rst

```diff
@@ -0,0 +1,152 @@
+.. _test_organization:
+
+Test organization
+=================
+Ideally, there should be one, and only one, obvious place for a test to reside.
+Until we reach that ideal, these are some rules of thumb for where a test should
+be located.
+
+1. Does your test depend only on code in ``pd._libs.tslibs``?
+This test likely belongs in one of:
```

Comment:
This line needs to be aligned with the "Does .." from the line above. See eg https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks for rst syntax for lists (and nested lists)